.ci/setenvconfig

```diff
@@ -392,6 +402,7 @@ BUILD_NUMBER=$BUILD_NUMBER
 E2E_PROVIDER=gke
 CLUSTER_NAME=eck-pr-$BUILD_NUMBER
 TEST_OPTS=-race
+E2E_TEST_LOG_LEVEL=1
```

Comment:
This variable is not the same you used here: 

https://github.com/elastic/cloud-on-k8s/pull/4098/commits/d36d8d97aaa66c2017f3716789d8b0817c63b9d4#diff-80ee1228a1e82f790f5ca21136914b25bc21a9d7af72a212ada108bb6e4dd53cR34

Reply:
Indeed! 🤦🏼